src/main/kotlin/blackjack/domain/CardNumberCalculator.kt

```diff
@@ -0,0 +1,34 @@
+package blackjack.domain
+
+import blackjack.domain.card.Card
+import blackjack.domain.card.CardNumber
+
+class CardNumberCalculator {
+
+    fun calculateSumOfCardNumbers(cards: List<Card>): Int {
+        var sum = 0
+        cards.forEach {
+            sum += calculateCardNumber(it.number, sum)
+        }
+        return sum
+    }
+
+    fun calculateCardNumber(cardNumber: CardNumber, sum: Int): Int {
+        if (cardNumber == CardNumber.A) {
+            return proceedAceNumber(sum)
+        }
+        return cardNumber.value
+    }
+
+    fun proceedAceNumber(sum: Int): Int {
+        if (sum + ACE_MAXINUM <= RuleChecker.CONDITION_TO_WIN_BLACK_JACK) {
+            return ACE_MAXINUM
+        }
+        return ACE_MINIMUM
+    }
+
+    companion object {
+        const val ACE_MAXINUM = 11
+        const val ACE_MINIMUM = 1
```

Comment:
```suggestion
        private const val ACE_MAXIMUM = 11
        private const val ACE_MINIMUM = 1
```
클래스 내부에서만 접근하는 상수값이라면, private 접근자를 붙여보면 어떨까요?